examples/jsm/controls/OrbitControls.js

```diff
@@ -278,12 +279,15 @@ class OrbitControls extends EventDispatcher {
 
 				if ( zoomChanged ||
 					lastPosition.distanceToSquared( scope.object.position ) > EPS ||
-					8 * ( 1 - lastQuaternion.dot( scope.object.quaternion ) ) > EPS ) {
+					8 * ( 1 - lastQuaternion.dot( scope.object.quaternion ) ) > EPS ||
+					lastTargetPosition.distanceToSquared( scope.target ) > 0 ) {
```

Comment:
@mrdoob What was the reason you did not use EPS here?

Reply:
Comparing with EPS didn't fix the issue.

Reply:
I `console.log()`ed `lastTargetPosition.distanceToSquared( scope.target ) > 0`.

Seems to be fine:

https://github.com/mrdoob/three.js/assets/97088/822d571a-8fa3-4d4e-895a-875ab93ada63

Reply:
When damping is enabled, that branch is triggered 600 times when 0 is used -- instead of 30 times when EPS is used. But update() is called every frame, anyway, so in hindsight, it is a non issue.

Thanks.